test/integration/acceptance/grant_fixture.hpp

```diff
@@ -0,0 +1,411 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef IROHA_GRANT_FIXTURE_H
```

Comment:
Should be `IROHA_GRANT_FIXTURE_HPP`. Also, please, don't forget to fix the comment in the end of the file